test/fixtures/v5/turn/left_ref_default.json

```diff
@@ -0,0 +1,11 @@
+{
+    "step": {
+        "maneuver": {
+            "type": "turn",
+            "modifier": "left"
+        },
+        "name": "",
+        "ref": "Ref1;Ref2"
+    },
+    "instruction": "Turn left onto Ref1"
```

Comment:
@bsudekum @1ec5 @willwhite @karenzshea highlighting here that I've changed the behaviour to only include the first `ref`, instead of all. This is a debatable decision, but I value conciseness highly here, thus one ref should be sufficient. Choosing the best ref should be smarter though, e.g. via #30.